app/manifest.json

```diff
@@ -49,7 +50,9 @@
         "activeTab",
         "storage",
         "https://boilerpipe-web.appspot.com/*",
-        "https://mercury.postlight.com/*"
+        "https://mercury.postlight.com/*",
+        "http://*/",
```

Comment:
The problem with this change is that it increases the permissions required by the extension.
The extension will be disabled as soon as it is updated, waiting for the user to approve the new permissions.

I am worried that many people will uninstall it after seeing that the extension can access **Your data on all the websites you visit**